V0.0.2 display fee est

matt-auckland opened the pull request:
Follow up to https://github.com/NAVCoin/NavMorph/pull/216

Commit:
made the estimate fees method more generic so it can be used in other places. Will need to pulll it out of the data service at some point.

Commit:
we now display the fee estimate

red010b37 commented on frontend/src/app/pages/status/status.component.ts:
```diff
@@ -36,7 +36,8 @@ export class StatusPage implements OnInit {
     private GenericFuncs: GenericFunctionsService,
-    private router: Router
+    private router: Router,
+    private dataService: SendPageDataService
```
your service vars are not consistent - mix of upper a lower case.

also for private vars I prefer the _ notations 


e.g
 private OrderService: OrderService,

would be
 private _orderService: OrderService,

matt-auckland replied:
_ notation seems nice. I'll switch to it here. Won't be overnight but, I'll eventually propagate this notation style around the rest of the front end as well.

red010b37 replied:
Yea thats totally cool!

zanuka replied:
roger that

red010b37 reviewed (commented):
Left a note for you around consistency or the DI vars

Commit:
- removed genericfunctions as we were using it
- updated private funcs to use _thisConvention
- fixed a mising var issue I created when I refactored the getEstimate function
- updated various places to deal with the fixed getEstimate function

Commit:
forgot to access the data properly :facepalm:
